core/domain/wipeout_service.py

```diff
@@ -391,6 +408,12 @@ def delete_user(
 
     auth_services.delete_external_auth_associations(user_id)
 
+    # Remove profile picture.
+    user_settings_model = user_models.UserSettingsModel.get_by_id(user_id)
+    username = user_settings_model.username
+    if username is not None:
+        _delete_profile_picture(username)
```

Comment:
I think this can be moved under `if feconf.ROLE_ID_MOBILE_LEARNER not in user_roles:` and then we do not need the username is not None check.